repose-aggregator/artifacts/src/config/scripts/repose-valve-deb

```diff
@@ -33,18 +33,24 @@ CLEAN=/usr/bin/clean-repose-deploy
 # Can override the defaults in /etc/sysconfig
 . /etc/sysconfig/repose
```

Comment:
It should be by Debian convention, but it never has been.
The Redhat way has just always been forced on the Debian boxes.